profiles/TusharKesarwani.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/92527686/216423127-3227d434-3b7c-4a94-8d56-0ef8a72f6dc5.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/216521554-ecee93b0-f4c6-47a4-a48b-72a6cc77e966.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/216521554-ecee93b0-f4c6-47a4-a48b-72a6cc77e966.png",
```